traffic_ops/testing/api/v3/deliveryservice_requests_test.go

```diff
@@ -209,29 +405,43 @@ func TestDeliveryServiceRequestWorkflow(t *testing.T) {
 		}
 
 		alerts, dsr := updateDeliveryServiceRequestStatus(t, dsrs[0], "submitted")
+		found = false
+		for _, alert := range alerts.Alerts {
+			if alert.Level == tc.ErrorLevel.String() {
+				t.Errorf("Unexpected error-level alert: %s", alert.Text)
+			} else if alert.Level == tc.SuccessLevel.String() && strings.Contains(alert.Text, "updated") {
+				found = true
+			}
+		}
 
-		expected = []string{
-			"deliveryservice_request was updated.",
+		if !found {
+			t.Error("Didn't find success-level alert after updating")
 		}
 
-		utils.Compare(t, expected, alerts.ToStrings())
 		if dsr.Status != tc.RequestStatus("submitted") {
 			t.Errorf("expected status=submitted,  got %s", string(dsr.Status))
 		}
 	})
 }
 
-func updateDeliveryServiceRequestStatus(t *testing.T, dsr tc.DeliveryServiceRequest, newstate string) (tc.Alerts, tc.DeliveryServiceRequest) {
-	ID := dsr.ID
+func updateDeliveryServiceRequestStatus(t *testing.T, dsr tc.DeliveryServiceRequestV30, newstate string) (tc.Alerts, tc.DeliveryServiceRequestV30) {
+	if dsr.ID == nil {
+		t.Error("Cannot update DSR with no ID")
```

Comment:
Why not?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

> Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context.

Reply:
That's not an error string, it's a testing error, which is printed directly, not returned to the caller to be possibly appended to something else